universe/ShipPart.cpp

```diff
@@ -282,6 +286,8 @@ bool ShipPart::operator==(const ShipPart& rhs) const {
     CHECK_COND_VREF_MEMBER(m_production_cost)
     CHECK_COND_VREF_MEMBER(m_production_time)
     CHECK_COND_VREF_MEMBER(m_location)
+    CHECK_COND_VREF_MEMBER(m_total_fighter_damage)
```

Comment:
do these two actually affect how the part functions, or just the UI representation?

if not, it perhaps doesn't need to be checksummed...

Reply:
It is supposed to be used in UI and AI

Reply:
I guess if it's used in the AI then it can be included in the calculation